rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -98,13 +101,7 @@ class ReportDisplayer(
   def asyncDisplay(node : NodeInfo) : NodeSeq = {
     val id = JsNodeId(node.id)
     val callback =  SHtml.ajaxInvoke(() => SetHtml("reportsDetails",displayReports(node)) )
-    Script(OnLoad(JsRaw(
```

Comment:
I remember now. This code doesn't work since the bind function is deprecated (since Jquery 3.0.0) !